docs/modules/ROOT/partials/proc-persistence-kafkasql-scram.adoc

```diff
@@ -1,5 +1,5 @@
-[id="registry-persistence-kafka-streams-scram"]
-= Configuring Kafka Streams storage with SCRAM security
+[id="registry-persistence-kafkasql-scram"]
```

Comment:
The example at the start of this file shows a Kafka version of 2.5.0. This should be Kafka 2.7.0 (used by the supported version of AMQ Streams 1.7)